app/views/events/_afterparty.slim

```diff
@@ -0,0 +1,11 @@
+h3 Awesome afterparty with topic rooms! 🎉
+
+h2 Pick a topic you like and be engaged!
```

Comment:
```suggestion
h2 Five rooms, five different topics. Choose your favorite and hang out with fellow pivorakers!
```

Reply:
If this is too long and difficult to read, then simply `Five rooms, five different topics:`